ext/opentelemetry-ext-opencensus/src/opentelemetry/ext/opencensus/__init__.py

```diff
@@ -18,7 +18,7 @@
 .. code:: python
 
     from opentelemetry import trace
-    from opentelemetry.ext.otcollector.trace_exporter  import CollectorSpanExporter
+    from opentelemetry.ext.opencensus.trace_exporter  import CollectorSpanExporter
```

Comment:
Should the name of the exporters be updated to reflect this is specifically for the OpenCensus Collector as opposed to the OpenTelemetry collector? I don't know if it will create confusion to have multiple `CollectorSpanExporter` or if the package naming is differentiating enough.

Reply:
I fixed some naming in the last commit, please let me know if it is right now :+1: